tutorials/xr/xr_action_map.rst

```diff
@@ -0,0 +1,246 @@
+
+This generic name is chosen because controllers don't cover the entire system. Currently there are also profiles for trackers, remotes and tracked pens. There are also provisions for devices such as treadmills, haptic vests and such even though those are not part of the specification yet.
+
+.. warning::
+  It is important to know that OpenXR has strict checking on supported devices. The core specification identifies a number of controllers and similar devices with their supported inputs and outputs, every XR runtime must accept these interaction profiles even if they aren't applicable. New devices are added through extensions and XR runtimes must specify which they support. XR runtimes that do not support a device added through extensions will not accept these profiles. XR runtimes that do not support added input or output types will often crash if supplied.
+
+  As such Godot keeps meta data of all available devices, their inputs and outputs and which extension adds support for them. You can create interaction profiles for all devices you wish to support, Godot will filter out those not supported by the XR runtime the user is using.
+
+  This does mean that in order to support new devices, software changes are needed.
+
+It is however also important to note that the action map has been designed with this in mind. When new devices enter the market, or when your users use devices that you do not have access to, the action map system puts the onus with the XR runtime. It is the XR runtimes job to choose the best fitting interaction profile that has been specified and adapt it for the controller the user is using.
+
+How the XR runtime does this is left to the implementation of the runtime and there are thus vast differences between the runtimes.
+
+A common approach for a runtime is to look for a matching interaction profile first, if this is not found check the most common profiles such as that of the "Touch controller" and do a conversion, and only if all else fails, check the generic "Simple controller", more on that one in a bit.
+
+.. note::
+  There is an important conclusion to be made here. When a controller is found and the action map applied to it, the XR runtime will choose the most applicable interaction profile and enact the bindings. At this point it will communicate this choice to the game. We have to be careful that we do not assign a conclusion to this that can't be made.
+
+  For example, when the Touch controller profile is used any of the following scenarios could be true:
+
+   * we could be using a Quest 1 controller,
+   * we could be using a Quest 2 controller,
+   * we could be using a Quest Pro controller but no Quest Pro profile was given or the XR runtime being used does not support the Quest Pro controller,
+   * it could be a completely different controller for which no profile was given but the XR runtime is using the touch bindings as a base.
```

Comment:
```suggestion
    * it could be a completely different controller for which no profile was given but the XR runtime is using the touch bindings as a base.
```